brownie/_cli/analyze.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/python3
+
+import json
+import re
+import time
+from os import environ
+
+from docopt import docopt
+from mythx_models.response import Severity
+from pythx import Client
+from pythx.middleware.toolname import ClientToolNameMiddleware
+
+from brownie import project
+from brownie._cli.__main__ import __version__
+from brownie._config import ARGV, _update_argv_from_docopt
+from brownie._gui import Gui
+from brownie.exceptions import ProjectNotFound
+
+__doc__ = f"""Usage: brownie analyze [options] [--async | --interval=<sec>]
+
+Options:
+  --gui                   Launch the Brownie GUI after analysis
+  --full                  Perform a full scan (MythX Pro required)
+  --interval=<sec>        Result polling interval in seconds [default: 3]
+  --async                 Do not poll for results, print job IDs and exit
+  --access-token          The JWT access token from the MythX dashboard
```

Comment:
```suggestion
  --access-token <token>  The JWT access token from the MythX dashboard
```

need to include ``<token>`` or similar here for ``docopt`` to properly interpret the cli input.  Same for ``--eth-address`` and ``--password`` on the next lines.

Reply:
Already fixed in 79b3556